### DISCUSSION TITLE
How to implement "sequental" validator?

### DISCUSSION THREAD

**Discussion by kromar777** [making/yavi#256] [Q&A] [answered]
Hi

Suppose there is a String validator checking on input string
StringValidator<String> nameValidator = StringValidatorBuilder.of("serialnum", c -> c.fixedSize(16).isInteger()).build(); 
And then, if the string is parsable to int, I want to apply constraints to the parsed integer like lessThan or another.
How to to achieve it?

**Comment by making** [comment, marked as answer]
Same as https://github.com/making/yavi/issues/245#issuecomment-1118668743 ?

**Reply by kromar777** [reply]
Yes, thank you for pointing out!
